docs/data/graph.md

```diff
@@ -86,13 +86,13 @@ We can also identify several potential arcs in the Event. In order of obviousnes
 1. From the `subj_id`, `relation_type_id` and `obj_id` fields:
     - Content at the URL `https://reddit.com/r/science/comments/42p4xg/prognostic_indicators_for_ebola_patient_survival/`
     - discusses
-    - the Registered Content Item with DOI `https://doi.org/10.3201/eid2202.151250`
+    - the registered content item with DOI `https://doi.org/10.3201/eid2202.151250`
 2. From the `subj.url` and `obj.url` fields:
     - The content at the URL `https://reddit.com/r/science/comments/42p4xg/prognostic_indicators_for_ebola_patient_survival/`
     - referred to the subject
     - using URL `http://wwwnc.cdc.gov/eid/article/22/2/15-1250_article`
 3. From the `subj_id` field:
-    - The Event ID `0003a012-e3fd-4d2f-8c18-1d8b7bb07e20`
+    - the Event ID `0003a012-e3fd-4d2f-8c18-1d8b7bb07e20`
```

Comment:
These examples are meant to read as sentences, broken over three lines, hence sentence case. Maybe reconsider?